Gandalf/build.sh

```diff
@@ -26,61 +21,41 @@ PKG_AUTHOR="EthanRDoesMC <ethanrdoesmc@gmail.com>"
 PKG_REPLACES="com.enduniverse.cydiaextenderplus, com.github.ethanrdoesmc.gandalf, com.github.ethanrdoesmc.gandalf102"
 PKG_ARCHITECTURE='iphoneos-arm'
 
-# Check if everything is ok
+BOLD=$(tput bold)
+NORMAL=$(tput sgr0)
+RED=$(tput setaf 1)
 
-# Check if started without parameters
-if [ "$1" = "" ]; then
-   echo "${bold}USAGE:${normal} './build.sh <versionfolder>'"
-   exit 0
-fi
-
-# Check if folder exists
 
-if [ ! -d $1 ]; then
-   echo "${bold}${red}FATAL:${normal} There's no folder called '$1'. You have either misstyped something ${bold}(Linux is case-sensitive)${normal} or the folder '$1' simply doesn't exist."
-   echo "ABORT."
-   exit 1
+#Main script
+	
+#Sanity Checks
+if [ -z "$1" ]; then
+  echo "${BOLD}Usage:${NORMAL} './build.sh <version>"
+  exit 0
 fi
 
-# Check filestructure
-if [ ! -f $1/conflicts.txt ] || [ ! -f $1/firmware.txt ] || [ ! -f $1/name.txt ] || [ ! -f $1/section.txt ]; then
-   echo "${bold}${red}FATAL:${normal} Please check folder '$1'. There's either something missing or misstyped. Be sure you have the files"
-   echo
-   echo "  - conflicts.txt"
-   echo "  - firmware.txt"
-   echo "  - name.txt"
-   echo "  - section.txt"
-   echo
-   echo "in folder '$1'."
-
-    # Test if folder is empty
-    if test "$(ls -A "$1")"; then
-       echo "Currently there are the following files in folder '$1':"
-       echo "---"
-       ls -la $1
-       echo "---"
-       echo "${bold}Linux is case-sensitive!${normal}"
-     else
-       echo "${bold}ERROR:${normal} The folder '$1' is empty!"
-   fi
-
-   echo "${bold}ABORT.${normal}"
-   exit 1
+if [ ! -d "$1" ] || [ ! -f "$1/conflicts.txt" ] || [ ! -f "$1/firmware.txt" ] || [ ! -f "$1/name.txt" ] || [ ! -f "$1/section.txt" ]; then
+	echo "${BOLD}${RED}ERROR:${NORMAL} Please check if these files or folders exists"
+	echo 
```

Comment:
* folders exist.
I'm no native speaker (and I don't speak English very fluently) but I think it's wrongly spelled.